manifest-updater/build.sh

```diff
@@ -7,4 +7,6 @@ BUILD_ID="$(git symbolic-ref --short HEAD 2>/dev/null || true)+${COMMIT_ID}"
 BUILD_TIME="$(date)"
 BUILD_INFO="${BUILD_ID} - ${BUILD_TIME}"
 
-docker build --tag letsencrypt/prio-manifest-updater --build-arg BUILD_INFO="${BUILD_INFO}" .
+cd ..
+
+docker build -f manifest-updater/Dockerfile --tag letsencrypt/prio-manifest-updater --build-arg BUILD_INFO="${BUILD_INFO}" .
```

Comment:
nit: does this work?
```suggestion
docker build -f manifest-updater/Dockerfile --tag letsencrypt/prio-manifest-updater --build-arg BUILD_INFO="${BUILD_INFO}" ..
```
Seems a bit less stateful than `cd`.

Reply:
It does! Updated accordingly.

Reply:
Do you perhaps need to push your latest changes for them to show up here?

Reply:
oops, yup. sorry about that.